lib/vizkit/vizkit_items.rb

```diff
@@ -803,6 +803,27 @@ def write(&block)
         end
     end
 
+    class TaskModelItem < VizkitItem
+        attr_reader :model
+
+        def initialize(task, options = Hash.new)
```

Comment:
> since it is an attribute on VizkitItem?

Yeah ... I missed that one. Fair point. VizkitItem itself does not use the ivar, it should not have had it in the first place... :(